### PR TITLE
Add nsswitch.conf to our docker image

### PR DESCRIPTION
**Pull request by harshavardhana**
## Description
Add nsswitch.conf to our docker image

## Motivation and Context
This will cause the alpine images to resolve DNS 
using /etc/hosts along with the normal DNS resolver.

## How Has This Been Tested?
Manually setting /etc/hosts

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,5 +1,7 @@
 FROM alpine:3.5
 
+RUN echo 'hosts: files dns' >> /etc/nsswitch.conf
```
Comment (balamurugana) [inline Dockerfile]: Is it good idea to conditionally add/edit `hosts: files dns` than forceful append?

Comment (harshavardhana) [reply Dockerfile]: In-fact this file doesn't even exist @balamurugana inside the container. It is meant for only one purpose.

File: Dockerfile.aarch64 [hunk 2]
```diff
@@ -9,6 +13,7 @@ WORKDIR /go/src/github.com/minio/
 RUN  \
      apk add --no-cache ca-certificates && \
      apk add --no-cache --virtual .build-deps git go musl-dev && \
+     echo 'hosts: files mdns4_minimal [NOTFOUND=return] dns mdns4' >> /etc/nsswitch.conf && \
```
Comment (balamurugana) [inline Dockerfile.aarch64:14]: Two `RUN` have `/etc/nsswitch.conf` updated.  Is it expected?

File: Dockerfile.armhf [hunk 3]
```diff
@@ -9,6 +13,7 @@ WORKDIR /go/src/github.com/minio/
 RUN  \
      apk add --no-cache ca-certificates && \
      apk add --no-cache --virtual .build-deps git go musl-dev && \
+     echo 'hosts: files mdns4_minimal [NOTFOUND=return] dns mdns4' >> /etc/nsswitch.conf && \
```
Comment (balamurugana) [inline Dockerfile.armhf:14]: same as above.

**Review by balamurugana — CHANGES_REQUESTED** [review]

File: Dockerfile.aarch64 (hunk 2 above)
Comment (harshavardhana) [reply Dockerfile.aarch64:14]: Should be updated my mistake

**Review by nitisht — APPROVED** [review]

**Comment by harshavardhana** [comment]
@balamurugana can you review once again?

**Review by balamurugana — APPROVED** [review]

**Pull request merged**